docs/cloud-native-security/vuln-management-get-started.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[vuln-management-get-started]]
+= Get started with CNVM
+
+This page explains how to set up Cloud Native Vulnerability Management (CNVM).
+
+.Requirements
+[sidebar]
+--
+* CNVM is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* {stack} and {agent} versions 8.8 or higher.
+* To view vulnerability scan findings, you need at least `read` privileges for the following indices:
+** `logs-cloud_security_posture.vulnerabilities-*`
+** `logs-cloud_security_posture.vulnerabilities_latest-*`
+--
+
+NOTE: CNVM currently only supports AWS EC2 Linux workloads.
+
+[discrete]
+[[vuln-management-setup]]
+== Set up CNVM for AWS
+
+To set up the CNVM integration for AWS, install the integration on a new {agent} policy, sign into the AWS account you want to scan, and run the https://docs.aws.amazon.com/cloudformation/index.html[CloudFormation] template.
+
+IMPORTANT: Do not add the integration to an existing {agent} policy. It should always be added to a new policy since it should not run on VMs with existing workloads. For more information, refer to <<vuln-management-overview-how-it-works, Vulnerability Management overview>>.
+
+[discrete]
+[[vuln-management-setup-step-1]]
+=== Add the CNVM integration
+
+. In the {security-app}, go to the **Get started** page, then click *Add security integrations*.
+. Search for **Cloud Native Vulnerability Management**, then click on the result.
+. Click *Add Cloud Native Vulnerability Management*.
+. Give your integration a name that matches its purpose or the AWS account region you want to scan for vulnerabilities (for example, `uswest2-aws-account`.)
++
+image::images/cnvm-setup-1.png[The CNVM integration setup page]
++
+. Click *Save and continue*. The integration will create a new {agent} policy.
+. Click *Add {agent} to your hosts*.
+
+[discrete]
+[[vuln-management-setup-step-2]]
+=== Sign in to the AWS management console
+
+. Open a new browser tab and use it to sign into your AWS management console.
+. Switch to the cloud region with the workloads that you want to scan for vulnerabilities.
+
+IMPORTANT: The integration will only scan VMs in the region you select. To scan multiple regions, repeat this setup process for each region.
+
+[discrete]
+[[vuln-management-setup-step-3]]
+=== Run the CloudFormation template
+
+. Switch back to the tab where you have {kib} open.
+. Click *Launch CloudFormation*. The CloudFormation page appears.
++
+image::images/cnvm-cloudformation.png[The cloud formation template]
```

Comment:
This screenshot feels unnecessary — not because it's not our UI, but because the user is doing almost nothing on this screen other than just click through (with one optional config change). Even if this were a Kibana UI, I'd rethink whether the user needs a huge screenshot (and whether we want to maintain it) just to click a single, pretty conspicuous button.

Reply:
I tend to agree but I'll defer to @jmikell821 on this one, since I added the AWS image at her suggestion.